### PR TITLE
Re-enable VMS console filtering

### LINKED ISSUE #53: Fix surefire invocation in VMS
The commented-out surefire invocation for VMS works when eMOP is manually added to a project, but fails when an extension is used. Ideally, it should work in both cases.

### PR DESCRIPTION
**Pull request by bcarlet**
Resolves #53.

### CODE REVIEW

**Comment by bcarlet** [comment]
This PR is incompatible with #62. Whichever is merged second will need to be modified to accommodate the other.

File: emop-maven-plugin/src/main/java/edu/cornell/VmsMojo.java [hunk 1]
```diff
@@ -228,6 +228,8 @@ private boolean filter(String violation) {
     private void invokeSurefire() throws MojoExecutionException {
         InvocationRequest request = new DefaultInvocationRequest();
         request.setGoals(Collections.singletonList("surefire:test"));
+        request.setProperties(System.getProperties());
+        request.setBatchMode(true);
```
Comment (marques-valeria) [inline emop-maven-plugin/src/main/java/edu/cornell/VmsMojo.java:232]: Out of curiosity - what exactly was the problem that surefire was running into with the extension / how does this fix it?

Comment (bcarlet) [reply emop-maven-plugin/src/main/java/edu/cornell/VmsMojo.java:249]: From what I've observed, the problem wasn't with the extension. It was that some properties were being set on the command line (e.g., `maven.repo.local`) and these weren't being passed to Surefire. This change just forwards all the properties that VMS was run with to Surefire.

**Comment by marques-valeria** [comment]
lgtm

**Comment by ayakayorihiro** [comment]
Looks good to me, though it seems like the CI didn't work? Can you please look into why and fix?

**Comment by owolabileg** [comment]
Closing this PR that is now in conflict. It is also no longer clear to me that the PR is still needed. If someone is available to work on it, we can reopen.

**Pull request closed without merging**